app/(general)/integration/livepeer/page.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import { motion } from 'framer-motion'
+import Image from 'next/image'
+import Balancer from 'react-wrap-balancer'
+
+import { BranchColorMode } from '@/components/shared/branch-color-mode'
+import { LinkComponent } from '@/components/shared/link-component'
+import { Button } from '@/components/ui/button'
+import { FADE_DOWN_ANIMATION_VARIANTS } from '@/config/design'
+import { turboIntegrations } from '@/data/turbo-integrations'
+
+export default function LivePeerNav() {
+  return (
+    <div className="flex-center flex flex-1 flex-col items-center justify-center">
+      <motion.div
+        className="max-w-3xl px-5 text-center xl:px-0"
+        initial="hidden"
+        whileInView="show"
+        animate="show"
+        viewport={{ once: true }}
+        variants={{
+          hidden: {},
+          show: {
+            transition: {
+              staggerChildren: 0.15,
+            },
+          },
+        }}>
+        <BranchColorMode>
+          <Image className="mx-auto rounded-full" alt="Disco logo" src={turboIntegrations.livePeer.imgDark} width={100} height={100} />
+          <Image className="mx-auto rounded-full" alt="Disco logo" src={turboIntegrations.livePeer.imgLight} width={100} height={100} />
+        </BranchColorMode>
+        <motion.h1
+          className="text-gradient-sand my-8 text-center text-4xl font-bold tracking-[-0.02em] drop-shadow-sm md:text-8xl md:leading-[6rem]"
+          variants={FADE_DOWN_ANIMATION_VARIANTS}>
+          {turboIntegrations.livePeer.name}
+        </motion.h1>
+        <motion.p className="my-4 text-xl" variants={FADE_DOWN_ANIMATION_VARIANTS}>
+          <Balancer>{turboIntegrations.livePeer.description}</Balancer>
```

Comment:
livepeer was not added to the  `turboIntegrations` data object, you can find it [here](https://github.com/turbo-eth/template-web3-app/blob/main/data/turbo-integrations.ts)